taskvine/src/manager/vine_manager.c

```diff
@@ -501,6 +505,174 @@ static int handle_transfer_hostport(struct vine_manager *q, struct vine_worker_i
 	return VINE_MSG_PROCESSED;
 }
 
+void exit_alert(struct vine_manager *q, struct vine_worker_info *w, struct vine_task *t)
+{
+	if (t->result == VINE_RESULT_SUCCESS && t->time_workers_execute_last < 1000000) {
+		switch (t->exit_code) {
+		case (126):
+			warn(D_VINE,
+					"Task %d ran for a very short time and exited with code %d.\n",
+					t->task_id,
+					t->exit_code);
+			warn(D_VINE, "This usually means that the task's command is not an executable,\n");
+			warn(D_VINE, "or that the worker's scratch directory is on a no-exec partition.\n");
+			break;
+		case (127):
+			warn(D_VINE,
+					"Task %d ran for a very short time and exited with code %d.\n",
+					t->task_id,
+					t->exit_code);
+			warn(D_VINE, "This usually means that the task's command could not be found, or that\n");
+			warn(D_VINE, "it uses a shared library not available at the worker, or that\n");
+			warn(D_VINE, "it uses a version of the glibc different than the one at the worker.\n");
+			break;
+		case (139):
+			warn(D_VINE,
+					"Task %d ran for a very short time and exited with code %d.\n",
+					t->task_id,
+					t->exit_code);
+			warn(D_VINE, "This usually means that the task's command had a segmentation fault,\n");
+			warn(D_VINE, "either because it has a memory access error (segfault), or because\n");
+			warn(D_VINE, "it uses a version of a shared library different from the one at the worker.\n");
+			break;
+		default:
+			break;
+		}
+	}
+
+	vine_task_info_add(q, t);
```

Comment:
Move this one just after `exit_alert` is called? It does not really fit in the `exit_alert` function.

Reply:
`exit_alert` is removed as it was not called.

Reply:
Right, I think the pr moved that code to the function, and then the function was never called.